src/IrrationalConstants.jl

```diff
@@ -26,8 +26,12 @@ export
     logten,     # log(10)
     logπ,       # log(π)
     log2π,      # log(2π)
-    log4π       # log(4π)
+    log4π,      # log(4π)
+    invℯ,       # 1 / ℯ
+
+    LambertW_Ω  # Ω exp(Ω) = 1
```

Comment:
We should stick with the convention in this package and in base, and use lowercase names and in particular no camel case. I.e., e.g. `lambertwω` or just `ω`. Again, also we might want to replace it with or at least add an ASCII alias.

Reply:
Since it is called "Omega constant" I think we should just use `Ω` (or `ω` if we want to stick with the convention in Base and this package of only using lowercase names). And correspondingly, the ASCII alias would just be `Omega` or `omega`.

Reply:
In the revised version it's called `lambertw_Ω` and `lambertw_Omega` alias is added. I think plain `Ω` would be confusing, since it's less well known and/or may interfere with variable name in the user code.
`lambertwΩ` IMO is quite unreadable.

Reply:
All of the constants in `MathConstants` and `IrrationalConstants` follow a naming convention. The omega constant should follow the same convention. But, in any case, it should stay in `LambertW.jl`.